### PR TITLE
Update the style of the transcript to match the design doc.

### PR DESCRIPTION
**Pull request by amuamushu**
The transcript is now updated to reflect the transcript layout on the design document. In this update, users are able to manually scroll. Automatic scrolling with the video will be implemented later.
https://screenshot.googleplex.com/R81F3w57V1j.png

### CODE REVIEW

File: src/main/webapp/transcript.js [hunk 1]
```diff
@@ -56,27 +57,28 @@ function addMultipleTranscriptLinesToDom(transcriptLines) {
...
 
-  liElement.startDate = startDate;
-  liElement.endDate = endDate;
+  const liElement = document.createElement('li');
+  liElement.classList.add('align-self-center', 'mb-2');
+  liElement.appendChild(contentDivElement);
+  const hrElement = document.createElement('hr');
+  hrElement.classList.add('my-1', 'align-middle', 'mr-5');
+  liElement.appendChild(hrElement);
+  ulElement.appendChild(liElement);
```
Comment (n-wach) [inline src/main/webapp/transcript.js:81]: This section is a little hard to follow.  If the transcript styling gets more complicated (like when we add references/icons for comments) we might want to consider making this a customElement using a template

**Comment by jaysaleh** [comment]
Please resolve liner errors before merging

File: src/main/webapp/transcript.js (hunk 1 above)
Comment (amuamushu) [reply src/main/webapp/transcript.js:81]: I created an issue (#122) for this and will get to it later. Thanks for the tip!

**Comment by amuamushu** [comment]
> Please resolve linter errors before merging

Thanks for noticing. The linter issue is from patching something from a different branch. It should be resolved once it is merged into master.

File: src/main/webapp/transcript.js [hunk 2]
```diff
@@ -102,3 +104,14 @@ function appendParagraphToContainer(text, container, classes = []) {
 function deleteTranscript() {
   fetch('/delete-transcript', {method: 'POST'});
 }
+
+/**
+ * Coverts a Date representing a video {@code timestamp} into a string.
+ */
+// TODO: Move function to timestamp.js
+function timestampToString(timestamp) {
+  const ts = new Date(timestamp);
```
Comment (billie-cat) [inline src/main/webapp/transcript.js:113]: Please do not use abbreviations. Also this is a date, not a timestamp, so a better name would be date.

"Use the shortest possible name, but not at the cost of clarity. Avoid acronyms, abbreviations, and in-team jargon."

Please look at: https://docs.google.com/document/d/1_ldClEZLsI1cMpQl_BfeIlkjTsE2IiJ2PgvhABfO6vs/edit

File: src/main/webapp/transcript.js [hunk 3]
```diff
@@ -102,3 +104,14 @@ function appendParagraphToContainer(text, container, classes = []) {
...
+  const seconds = ts.getSeconds().toString().padStart(2, '0');
```
Comment (billie-cat) [inline src/main/webapp/transcript.js:114]: for 2 and '0' can you add the comments for your argument literals

https://docs.google.com/document/d/1-mH8Tg22ioPqe2x6_fjbUml8KkpbO0-KuSDLaql2VK4/edit#

File: src/main/webapp/style.css [hunk 4]
```diff
@@ -107,6 +107,18 @@ html {
   width: 50%;
 }
 
+/* Transcript */
```
[the remark this answers is not in the document]
Comment (amuamushu) [reply src/main/webapp/style.css]: Yes, I was waiting for #133 to be merged into master to make that change since the `lecture-view.css` file doesn't exist in master yet. I added file and moved the code over for now. 

Done! 

Edit: It was just added so I resolve the merge conflict.

File: src/main/webapp/transcript.js (hunk 2 above)
Comment (amuamushu) [reply src/main/webapp/transcript.js]: Thanks for the info!
This was a patch from an different pull request that I added in to ensure this pull request compiles. 

I believe it is now fixed and added to timestamp.js instead.

File: src/main/webapp/transcript.js (hunk 3 above)
Comment (amuamushu) [reply src/main/webapp/transcript.js]: This wasn't fixed in the other pull request I patched from so I just made the change. Thanks!

File: src/main/webapp/transcript.js (hunk 2 above)
Comment (billie-cat) [reply src/main/webapp/transcript.js]: I see, I would recommend not doing. If you absolutely must please add a review comment indicating this is not your code or else I end up reviewing the code twice.

Comment (amuamushu) [reply src/main/webapp/transcript.js]: Will do, sorry about that!
